plugin/build.gradle.kts

```diff
@@ -0,0 +1,76 @@
+@file:Suppress("UnstableApiUsage")
+
+import com.aliucord.gradle.AliucordExtension
+import com.android.build.gradle.LibraryExtension
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+import org.jlleitschuh.gradle.ktlint.KtlintExtension
+
+// TODO: Change to your repository
+val repo = "Aliucord/plugins-template"
```

Comment:
```suggestion
const val repo = "Aliucord/plugins-template"
```

Reply:
you cant use const in gradle script here